pyccel/codegen/cwrapper.py

```diff
@@ -81,7 +83,16 @@ def create_c_setup(mod_name,
     linker_flags_str   = ('extra_link_args = {0}'.format(print_list(linker_flags))
                    if flags else None)
 
-    args = [mod, wrapper_file, files, include_str, libs_str, libdirs_str, flags_str, linker_flags_str]
+    numpy_max_acceptable_version = [1, 19]
```

Comment:
Maybe it would make more sense to mark the last tested version of numpy in ast/numpywrapper.py?